packages/suite/src/views/settings/SettingsGeneral/Experimental.tsx

```diff
@@ -104,14 +108,21 @@ export const Experimental = () => {
                     }
                 />
                 <ActionColumn>
-                    <Switch isChecked={!!features} onChange={onSwitchExperimental} />
+                    <Switch
+                        isChecked={enabledFeatures !== undefined}
+                        onChange={onSwitchExperimental}
+                    />
                 </ActionColumn>
             </SectionItem>
             <AnimatePresence>
-                {features && Object.keys(ExperimentalFeature).length && (
+                {enabledFeatures && experimentalFeatures.length > 0 && (
```

Comment:
You really like unnecessary characters, don't you?

Reply:
I do not like unnecessary characters. What I do like is precision and consistency.

Reply:
It's perfectly consistent if you do it everywhere! Are you afraid of negative-length anti-arrays? :ghost:

Reply:
> It's perfectly consistent if you do it everywhere!

Is a path. :)

> Are you afraid of negative-length anti-arrays? 👻

I am afraid of future changes, that change left part of the expression to something that would (by luck) satisfy the condition. Precision here reduces (not eliminates) the surface for such a mistake.

It's just defensive programming. 

I also like concept of atomic habits. So I don't need to think and pay attention to every detail and think about all possible future scenarios. I am just strict in comparison everywhere.

Reply:
When empty or invalid js arrays will start to return truthy lengths, we'll have bigger problems than that. But whatever. :)

Reply:
I meant that whole `experimentalFeatures.length` will be replaced with something. And instead of a type error you get nothing.

Reply:
Here its pedantic, but in general this habit proven to me to be extremely useful. And because I don't want to think about when to do it and when not. I do it everywhere as it is super minor change to do and I do it automatically.